kickstarts/partials/post/python_modules.ks.erb

```diff
@@ -5,3 +5,167 @@ pip install vspk==5.3.2
 
 # For Lenovo
 pip install pylxca==2.1.1
+
+# For embedded ansible
+pip install virtualenv
+
+mkdir -p /var/lib/manageiq/
+pushd /var/lib/manageiq
+  virtualenv venv
+  source venv/bin/activate
+
+  cat > requirements.txt << EOF
```

Comment:
Minor, but would this be better as a straight file in COPY?

Reply:
I considered that, but thought that it might be more natural to keep what is essentially a packages list in this repo rather than in `manageiq-appliance`.